src/components/views/right_panel/UserInfo.tsx

```diff
@@ -510,7 +510,9 @@ export const isMuted = (member: RoomMember, powerLevelContent: IPowerLevelsConte
         (powerLevelContent.events ? powerLevelContent.events["m.room.message"] : null) ||
         powerLevelContent.events_default;
 
-    if (!levelToSend) return false;
+    // levelToSend could be undefined as .events_default is optional. Coercing in this case using
+    // Number() would always return false, so this preserves behaviour
```

Comment:
might be nice to stick a FIXME or something in here:

```suggestion
    // Number() would always return false, so this preserves behaviour
    // FIXME: per the spec, if `events_default` is unset, it defaults to zero. If
    //   the member has a negative powerlevel, this will give an incorrect result.
```